AnnotatorCore.py

```diff
@@ -257,6 +290,8 @@ def processalterationevents(eventfile, outfile, previousoutfile, defaultCancerTy
                 log.info("Cancer type for all samples must be defined\nline %s: %s" % (i, row))
                 # continue
 
+            conversion(hgvs)
```

Comment:
```suggestion
            hgvs = conversion(hgvs)
```
@darasanchez could you make the last change here?

Reply:
After converting the string, you need to reassign the variable